shipment_advice/models/shipment_advice.py

```diff
@@ -175,8 +175,9 @@ def _default_warehouse_id(self):
         ),
     )
     run_in_queue_job = fields.Boolean(
-        related="company_id.shipment_advice_run_in_queue_job"
+        default=lambda self: self._default_run_in_queue_job()
```

Comment:
Maybe tests are failing because the shipment advice used in the test is created before the flag is set on the company.
With a related field this is not an issue, the returned value will be the one currently set on the company, but with this change, the value is set at the shipment advice creation (and it's fine like this).

Here we enable the `run_in_queue_job` option in the test, but the shipment is created before in `setUpClass`:
https://github.com/OCA/stock-logistics-transport/blob/16.0/shipment_advice/tests/test_shipment_advice_async.py#L13-L14
I would change this line to:
```
self.shipment_advice_in.run_in_queue_job = True
self.shipment_advice_out.run_in_queue_job = True
```